README.md

```diff
@@ -10,7 +10,7 @@
 
 ## Architecture
 
-![architecture](./docs/media/cdc_architecture.png)
+<img src="docs/media/cdc_architecture.svg?sanitize=true" alt="architecture" width="600"/>
```

Comment:
`docs/design/2020-03-04-ticdc-design-and-architecture-cn.md` also need to update the image link